spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/list.html

```diff
@@ -173,14 +167,14 @@ <h2 th:text="#{admin.api-keys.subtitle}">[Post Administration</h2>
             </div>
         </div>
         <div th:include="admin/fragments/scripts :: scripts" th:remove="tag">Javascript Libraries</div>
-        <div th:include="admin/fragments/scripts :: initializeDataTables" th:remove="tag">Javascript Libraries</div>
+        <div th:include="admin/fragments/addOrderingToList :: addOrderingToAdminApiKeyList" th:remove="tag">Javascript Libraries</div>
         <div th:include="admin/fragments/scripts :: apiKeyRemoveDialog" th:remove="tag">Javascript Libraries</div>
 
     <script>
-        function createInfoBoxMessageAndDeleteKey(apiKeyId, consumerName){
-            infoBoxMessage = "Are you sure you want to delete the API key of <b>" + consumerName +
-                "</b>? Then, <b>" + consumerName + "</b> can not access this service anymore.";
-            deleteItem(apiKeyId, infoBoxMessage);
+        function createWarningBoxAndAskForPermissionToDeleteApiKey(apiKeyId, consumerName){
```

Comment:
As this not only asks for permission but also does the actual deletion, maybe rename it like this?
```
        function askForPermissionThenDeleteApiKey(apiKeyId, consumerName){
```
or
```
        function deleteApiKeyIfUserGivesPermission(apiKeyId, consumerName){
```

Reply:
Done. I preferred the second suggestions.